gw_spaceheat/input_data/houses.json

```diff
@@ -468,8 +467,7 @@
                     "RoleGtEnumSymbol": "57b788ee",
                     "DisplayName": "30A Relay for first boost element",
                     "ShNodeId": "e077df5d-7aea-4b9c-bfd7-773c1e65bd65",
-                    "ComponentId": "57e2eb41-08f4-4032-8948-b14890fce9ca",
-                    "HasActor": true
```

Comment:
This is handled by `sh_node.py` (part of my sloppy push adding the concept of ActorClass)
`property
    def has_actor(self) -> bool:
        if self.actor_class == ActorClass.NONE:
            return False
        return True
`